pkg/builtinchecks/yamls/sysctls.yaml

```diff
@@ -0,0 +1,9 @@
+name: "unsafe-sysctls"
+description: "Alert on deployments with unsafe allocation of sysctls"
+remediation: "Ensure containers does not allow unsafe allocation of CPU resources."
```

Comment:
Again, make these more descriptive, and link to documentation if possible.

Reply:
updated and added links.